network/p2pNetwork.go

```diff
@@ -471,6 +471,10 @@ func (n *P2PNetwork) meshThread() {
 
 func (n *P2PNetwork) httpdThread() {
```

Comment:
I suggest do not start this thread instead since `Start()` looks more appropriate place to check settings

Reply:
check out updates